kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -200,6 +196,11 @@ const List: React.FC<TopicsListProps> = ({
         setRecreateTopicConfirmationVisible,
       ] = React.useState(false);
 
+      const [
```

Comment:
maybe we can use useModal hook here , cause it is more declarative

Reply:
Or maybe I'll refactor all of them (in this component) with this PR.